Cython/Plex/Scanners.py

```diff
@@ -337,3 +339,4 @@ def eof(self):
         Override this method if you want something to be done at
         end of file.
         """
+        pass
```

Comment:
Is this really needed?

Reply:
No, but I prefer to make it explicit.  Can revert if you have a strong preference.

Reply:
No, just wondering if that was something that flake8 would complain about.